python-sdk/tutorials/nuimages_tutorial.ipynb

```diff
@@ -184,10 +184,12 @@
     "- vehicles: orange\n",
     "- bicycles and motorcycles: red\n",
     "- pedestrians: blue\n",
-    "- cones and barriers: black\n",
+    "- cones and barriers: gray\n",
```

Comment:
Upon checking i think barriers are gray and cones are green gray

Reply:
Yeah, every single class has a different color. This is the quick summary.